Added the dynamic rendering of duotoned image feature

sohanpatharla opened the pull request:
Added a duotoned image effect feature for the about section, where images are processed and displayed with custom light and dark colors for dark and light themes.

### Changes:
- Added DuotoneImage component.
- Integrated it into the homepage to process portfolio images dynamically.

### How to Test:
- Visit the homepage and verify that the images are rendered with the duotone effect.
- Test the site on both light and dark themes.

sohanpatharla committed:
Added the dynamic rendering of duotoned image feature in the about section

BUMBAIYA commented:
Hi @sohanpatharla,

Great work on this! Thank you for contributing.

- Please rename the file `DuotoneImage.tsx` to `duotone-image.tsx` to follow the kebab-case convention.
- Remove the commented code from `about-hero.tsx` file.

Once these changes are made, I'll be happy to merge this PR.

Thanks again!

sohanpatharla committed:
Renamed DuotoneImage.tsx to duotone-image.tsx and removed commented code from about-hero.tsx

sohanpatharla commented:
Hey @BUMBAIYA ,

I’ve made the requested changes:

- Renamed `DuotoneImage.tsx` to `duotone-image.tsx` to follow the kebab-case convention.
- Removed the commented code from the `about-hero.tsx` file.

The updates are now ready for review.

:)

sohanpatharla committed:
minor fix:Update the dependency array following linting rules
